### PR TITLE
feat: support for adding, removing, updating machines/devices Workload Annotations

### PR DESCRIPTION
**Pull request by sachin120**
#### What does this implement

add, remove or update Machine/Device workload annotations

### CODE REVIEW

**Comment by Sovietaced** [comment]
It would also be helpful to read workload annotations from machines/devices. This would be required for the terraform provider to work correctly with this feature.

**Comment by sachin120** [comment]
> It would also be helpful to read workload annotations from machines/devices. This would be required for the terraform provider to work correctly with this feature.

I have added the `WorkloadAnnotation` Field in both structs Machine/Device. This should fix getting values from the Machine/Device. Is this correct ??

@Sovietaced I am not able to fix Go-Lint errors. Can you pls guide me?

File: client/device.go
Comment (skatsaounis) [inline client/device.go:1]: Just remove line 1 here.

File: entity/device.go [hunk 1]
```diff
@@ -3,22 +3,23 @@ package entity
 import "net"
 
 type Device struct {
```
Comment (skatsaounis) [inline entity/device.go:5]: This gist can help you fix this issue: https://gist.github.com/up1/69a6749df9464f15743dbb1a0c711762

File: entity/machine.go [hunk 2]
```diff
@@ -65,32 +65,33 @@ type Machine struct {
 			LinkID    int    `json:"link_id,omitempty"`
```
Comment (skatsaounis) [inline entity/machine.go:65]: Same as `entity/device` go suggestion

**Review by skatsaounis — COMMENTED** [review]
I can help you with the linter. Please take a look at the inline comments :slightly_smiling_face:

File: entity/device.go (hunk 1 above)
Comment (sachin120) [reply entity/device.go:5]: @skatsaounis It worked thanks.

**Review by skatsaounis — APPROVED** [review]
LGTM +1. Thank you for keeping up with the contributions. They are really welcomed :100:

**Pull request merged**